devicetypes/smartthings/aeon-multisensor-gen5.src/aeon-multisensor-gen5.groovy

```diff
@@ -12,14 +12,17 @@
  *
  */
 metadata {
-	definition (name: "Aeon Multisensor Gen5", namespace: "smartthings", author: "SmartThings") {
+	definition (name: "Aeon Multisensor Gen5", namespace: "smartthings", author: "SmartThings", category: "C3") {
 		capability "Motion Sensor"
 		capability "Temperature Measurement"
 		capability "Relative Humidity Measurement"
 		capability "Illuminance Measurement"
 		capability "Configuration"
 		capability "Sensor"
 		capability "Battery"
+		capability "Health Check"
+
+		command "configureAfterSecure"
```

Comment:
That's fine. I'm of the opinion that configure tiles should be taken out of all z-wave devices (see DVCSMP-264), but for now this is good.